.gitlab/pipeline/jobs/unit-php8.0-v11.yml

```diff
@@ -1,10 +1,10 @@
-unit-php7.3-v10:
+unit-php8.0-v11:
   extends: .default
-  image: php:7.3
+  image: php:8.0
   stage: test
   needs:
     - build-composer-dependencies
-    - php-lint-php7.3
+    - php-lint-php8.0
   script:
-    - composer require --no-progress typo3/minimal:"^10.4"
+    - composer require --no-progress typo3/minimal:"^11.5"
```

Comment:
Should this not be `typo3/cms-core` instead ? (#530 for example) ? Not rebased correctly ? Or mabye slipped through there.